### DISCUSSION TITLE
Doc文档无法预览

### DISCUSSION THREAD

**Discussion by m0eak** [lobehub/lobehub#5248] [General Question]
![image](https://github.com/user-attachments/assets/c9b2bb15-32a7-487e-900b-dab9387315cd)
显示如图

**Comment by arvinxx** [comment]
转成docx 再试试吧

**Reply by m0eak** [reply]
就是docx打不开也是这样显示，才转的doc

**Reply by m0eak** [reply]
![image](https://github.com/user-attachments/assets/aa937b78-b9ac-4597-9eaa-13ade3887c12)

**Reply by m0eak** [reply]
使用的是私有化部署的数据库版v1.42.1

**Reply by arvinxx** [reply]
咦… 方便这个文件发我吗？我晚点测试下

**Reply by m0eak** [reply]
> 咦… 方便这个文件发我吗？我晚点测试下

不是一个文档，我这里好像所有的word都预览不了

**Reply by m0eak** [reply]
> 咦… 方便这个文件发我吗？我晚点测试下

pdf 正常

**Reply by lovely009** [reply]
我也是一样的问题，有解决办法了吗?
